docroot/themes/humsci/humsci_basic/src/scss/utilities/_wysiwyg-text-area.scss

```diff
@@ -188,10 +188,6 @@
   max-width: hb-calculate-rems(700px);
 }
 
-.hs-well {
-  @include hb-well;
-}
-
```

Comment:
.hs-well still works testing within the editor and on content pages. Should be good here.